[2021-10-16 13:15:03,711] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T13:12:00+00:00 [queued]>
[2021-10-16 13:15:03,737] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T13:12:00+00:00 [queued]>
[2021-10-16 13:15:03,737] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:15:03,737] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-16 13:15:03,737] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:15:03,836] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_gross_year_1> on 2021-10-16 13:12:00+00:00
[2021-10-16 13:15:03,850] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'get_gross_year_1', 'scheduled__2021-10-16T13:12:00+00:00', '--job-id', '591', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dag.py', '--cfg-path', '/tmp/tmpnqzio68v', '--error-file', '/tmp/tmpuc5yklys']
[2021-10-16 13:15:03,851] {standard_task_runner.py:77} INFO - Job 591: Subtask get_gross_year_1
[2021-10-16 13:15:03,845] {standard_task_runner.py:52} INFO - Started process 1543 to run task
[2021-10-16 13:15:03,918] {logging_mixin.py:109} INFO - Running <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T13:12:00+00:00 [running]> on host e03fb9e01fd8
[2021-10-16 13:15:03,998] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fourth_dag
AIRFLOW_CTX_TASK_ID=get_gross_year_1
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T13:12:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T13:12:00+00:00
[2021-10-16 13:15:04,011] {python.py:152} INFO - Done. Returned value was: ['$42.05 ', '2010\n']
[2021-10-16 13:15:04,046] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=get_gross_year_1, execution_date=20211016T131200, start_date=20211016T131503, end_date=20211016T131504
[2021-10-16 13:15:04,109] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-16 13:15:04,170] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
